opm/io/eclipse/ESmry.hpp

```diff
@@ -39,6 +39,8 @@ class ESmry
 
     const std::vector<std::string>& keywordList() const { return keyword; }
 
+    int miniStepIdxAtReportStep(const int reportStep) const;
```

Comment:
1. Do we need to make this public?
2. The concept of "MINISTEP" is an Eclipse construction - if we need to export this to the public api - maybe we could rather use the term `timestep`?
3. `....AtReportStep`- implies at the *start* of the report step - or the end?

Reply:
It plays a role similar to libecl's `ecl_sum_iget_report_end()` function which in turn is used in in `test_msim_ACTIONX`.  If we don't make this&mdash;or something very close to this&mdash;into a public operation, then there's no easy way of replicating the `UDA` unit test.

I'm happy to rename the function.  At present it does return the ministep/timestep index at the start of the report step.

Reply:
Still a bit unclear to me whether this is the ministep at the beginning or the end of the report step; I don't really have a good alternative name, but if possible I would like a name with the following characteristics:

1. string `ministep` is removed.
2. Clear if we are at the beginning or end of report step.

You can do it!